landing.css

```diff
@@ -129,12 +142,47 @@ nav ul li {
 
   .description__cards {
     grid-template-columns: repeat(2, 1fr);
-    margin: 0 auto;
-    padding-right: 2rem;
+    gap: 2rem;
+    margin-bottom: -8rem;
+    margin-top: 6rem;
+  }
+
+  .description__card {
+    max-height: 11rem;
   }
 
   .description__goals {
     flex-direction: column;
     padding-top: 20rem;
   }
 }
+/** 
+* SECTION - (Third section)
+**/
+.section-three > div {
+  display: flex;
+  justify-content: space-around;
+  text-align: center;
+  flex-wrap: wrap;
+}
+
+.section-three > div > div {
+  flex-basis: 350px;
+}
+
+.section-three h2,
+.section-three p {
+  padding: 2rem;
+}
+
+/** 
+* COLORS
+**/
+
+.bg-black {
+  background: #2f2e41;
+}
+
+.bg-blue {
+  background: #1f59bb !important;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/gh/codeclannigeria/CodeClanUI/pullRequest?prid=5575170)